packages/thegraph-subgraph/package.json

```diff
@@ -2,6 +2,10 @@
     "name": "@dataunions/thegraph-subgraph",
     "version": "3.0.0-beta.1",
     "description": "Subgraph definitions for sidechain data unions",
+    "private": true,
```

Comment:
I wonder why this is private? I guess it's not even published to npm? 🤷🏻

Reply:
right, we don't want to make a package out of it, so "private" prevents publishing (even by accident)